broker/src/main/java/org/apache/rocketmq/broker/schedule/ScheduleMessageService.java

```diff
@@ -250,7 +250,6 @@ public boolean correctDelayOffset() {
                 }
                 if (correctDelayOffset != currentDelayOffset) {
                     log.error("correct delay offset [ delayLevel {} ] from {} to {}", delayLevel, currentDelayOffset, correctDelayOffset);
-                    offsetTable.put(delayLevel, correctDelayOffset);
```

Comment:
If it is not put in the table, the correction will have no effect.

Reply:
So there should be no reloading of the `delayOffset.json` file here, right?